resources/config.yaml

```diff
@@ -102,7 +102,7 @@ openml:                # configuration namespace for openML.
 
 versions:              # configuration namespace for versions enforcement (libraries versions are usually enforced in requirements.txt for the app and for each framework).
   pip:
-  python: 3.9          # the Python minor version that will be used by the application in containers and cloud instances, also used as a based version for virtual environments created for each framework.
+  python: 3.11          # the Python minor version that will be used by the application in containers and cloud instances, also used as a based version for virtual environments created for each framework.
```

Comment:
Is the framework not 3.9 compatible? Changing this number here will affect all frameworks. While we will raise this over time (and also plan to allow framework-specific definitions for this), we can't currently bump this without ensuring the compatibility for all other frameworks.